commit/chainfee/outcome.go

```diff
@@ -0,0 +1,130 @@
+package chainfee
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/types"
+	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+	"golang.org/x/exp/maps"
+
+	"github.com/smartcontractkit/chainlink-ccip/internal/libs/mathslib"
+	"github.com/smartcontractkit/chainlink-ccip/internal/plugincommon"
+	"github.com/smartcontractkit/chainlink-ccip/internal/plugintypes"
+)
+
+func (p *processor) getConsensusObservation(
+	aos []plugincommon.AttributedObservation[Observation],
+) (ConsensusObservation, error) {
+	aggObs := aggregateObservations(aos)
+
+	fMin := mathslib.RepeatedF(func() int { return p.bigF }, maps.Keys(aggObs.FChain))
```

Comment:
Without reading `RepeatedF`, I don't understand what's going on here.

Reply:
Hopefully this is a little clearer
```suggestion
	fMin := consensus.MakeMultiThreshold(fChains, consensus.TwoFPlus1)
```